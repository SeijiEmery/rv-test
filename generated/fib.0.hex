0000000    97  00  00  00  e7  80  00  00  6f  00  40  04  93  02  10  00
0000020    63  dc  a2  02  13  01  81  ff  93  02  f5  ff  23  30  51  00
0000040    13  05  e5  ff  97  00  00  00  e7  80  00  00  83  32  01  00
0000060    23  30  a1  00  33  05  50  00  97  00  00  00  e7  80  00  00
0000100    83  32  01  00  33  05  55  00  67  80  00  00  73  00  10  00
0000120
