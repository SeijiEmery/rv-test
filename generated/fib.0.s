call fib
j end
fib:
addi t0, zero, 1
ble a0, t0, return
addi sp, sp, -8
addi t0, a0, -1
sd t0, 0(sp)
addi a0, a0, -2
call fib

ld t0, 0(sp)
sd a0, 0(sp)
add a0, zero, t0
call fib

ld t0, 0(sp)
add a0, a0, t0
return:
ret
end:
ebreak
