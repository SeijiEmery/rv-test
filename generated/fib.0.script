load /x 0 generated/fib.0.hex
writereg 2 4096
writereg 10 10
run 0 1000
readreg 2
readreg 10