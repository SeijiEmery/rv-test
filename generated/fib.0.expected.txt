R2 = 4096
R10 = 55
